gensrc/thrift/PlanNodes.thrift

```diff
@@ -356,6 +356,9 @@ struct THdfsScanRange {
 
     // identity partition column slots
     18: optional list<Types.TSlotId> identity_partition_slot_ids;
+
+    // iceberg delete column slots
+    19: optional list<Types.TSlotId> equality_delete_slot_ids;
```

Comment:
iceberg_equality_delete_slot_ids
iceberg_delete_files

rename it?

Reply:
I have renamed it to delete_column_slots. not only for iceberg